apcd_cms/src/apps/utils/apcd_database.py

```diff
@@ -864,19 +900,19 @@ def get_cdl_exceptions(file_type):
             port=APCD_DB['port'],
             sslmode='require'
         )
-        query = """ SELECT c1.item_code as field_list_code, 
-        c1.item_value AS field_list_value, 
-        c2.item_value AS threshold_value 
-        FROM standard_codes c1 
-        LEFT JOIN standard_codes c2 
-            ON c2.item_code = c1.item_code 
-        WHERE c1.list_name=%s AND c2.list_name=%s """
+        query = """ SELECT  cdl_number as field_list_code, 
```

Comment:
I should have thought of this in my PR but it was so long ago since Chandra made this for us I forgot. Since we're touching this query, may as well implement Chandra's connection function to begin shortening this apcd_database util. We can replace all the conn declaration with this

```
   try:
        conn = db_connect()
        query = ...
```